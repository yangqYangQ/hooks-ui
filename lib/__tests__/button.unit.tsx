import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../button';

describe('button', () => {
    it('是一个 div', () => {
        let tree = renderer.create(<Button/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});