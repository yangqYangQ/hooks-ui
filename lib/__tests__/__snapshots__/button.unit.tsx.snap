// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 1`] = `
<div>
  这是一个 button
</div>
`;

exports[`button 是一个 div 1`] = `
<div>
  这是一个 button
</div>
`;
